index.rst

```diff
@@ -41,40 +41,38 @@ with version control systems.
 File Format
 ===========
 
-EditorConfig files use an INI format that is compatible with the format used
-by `Python configparser Library`_. In an EditorConfig file (usually named
```

Comment:
@xuhdev I forgot to mention that I removed this reference to the Python configparser library so that this spec can stand on its own two feet.

Reply:
Sounds good to me. Maybe we should delete them after we have included the AST?

Reply:
I’m not sure that it matters, as I’m not seeing any EC tests that actually support the many features of Python’s ConfigParser. There’s nothing breaking in removing this reference, as far as I can tell.

Reply:
Especially when you consider that all the non-Python cores are not using ConfigParser either.

Reply:
Sounds good